docs/processing/preprocessing.md

```diff
@@ -31,3 +31,43 @@ We avert that session-wise *fMRIPrep*'s processes run into race conditions by pr
+
+## Visualizing *fMRIPrep*'s individual reports
+
+Following our protocols<sup>[1]</sup>, the quality of unprocessed images MUST be assessed before and after preprocessing to verify it did not go awry.
+
+- [ ] Open each *fMRIPrep* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first section entitled `Summary` and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#summary).
+
+    !!! warning "Running through the visual report does not have to be executed in a fixed order"
+
+        While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related to the reportlet at hand. 
+        As such, we encourage you to jump back and forth between visualizations as much as needed.
+
+- [ ] Scroll down to the next section entitled `Anatomical conformation` and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#anatomical-conformation).
+- [ ] Scroll down to the first mosaic (brain mask and brain tissue segmentation) and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#brain-mask-and-brain-tissue-segmentation-of-the-t1w).
+- [ ] Scroll down to the dynamic visualization of the spatial normalization and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#spatial-normalization-of-the-anatomical-t1w-reference).
+- [ ] Scroll down to the next mosaic (surface reconstruction) and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#surface-reconstruction).
+- [ ] Scroll down to the functional section, starting with the textual summary and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#textual-summary).
+- [ ] Next, apply the [exclusion criteria](qaqc-criteria-preprocessed.md#t2-map) for the T2* map mosaic.
+- [ ] Scroll down to the histogram of T2* gray-matter values and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#t2-gray-matter-values).
+- [ ] Visualize the dynamic visualization of the co-registration by hovering your mouse on the image to make it toggle and removing it from the image for pausing. Apply the [exclusion criteria](qaqc-criteria-preprocessed.md#alignment-of-functional-and-anatomical-mri-data).
+- [ ] Visualize the next mosaic and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#brain-mask-and-anatomicaltemporal-compcor-rois).
+- [ ] Skip to the BOLD summary and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#bold-summary).
+- [ ] Scroll down to the heatmap and use it to [choose the regressors](qaqc-criteria-preprocessed.md#correlations-among-nuisance-regressors) you will include in the nuisance regression model.
+- [ ] Verify that no error is reported in the "Errors" section. 
+If there is, follow the procedure described in the appendage "Not all *fMRIPrep* derivatives were generated".
```

Comment:
```suggestion
If errors or quality issues are encountered, find the issue corresponding to that session in the dataset's repository and report a comprehensive description of the problems.
In case of *fMRIPrep* failure, follow the procedure described above "Not all *fMRIPrep* derivatives were generated".
```